healthCare-Frontend/src/components/assessment-form/step-three/StepThree.js

```diff
@@ -0,0 +1,291 @@
+import { useContext } from "react";
+import { Button, Col, Form, Row } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import DataContext from "../../../context/DataContext";
+import Stepper from "../../stepper/Stepper";
+import {
+  ATLEAST_ONE_SELECT,
+  REQUIRED_ERROR_MSG,
+} from "../../utils/constants";
+import Header from "../form-header/Header";
+import "../step-one/stepone.css";
+
+const StepThree = () => {
+  const { setData, handleNextClick, step } = useContext(DataContext);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const handleFormSubmit = (values) => {
+    console.log(values);
+    handleNextClick();
+    setData((prev) => ({
+      ...prev,
+      patient: {
```

Comment:
instead of `patient` property, it should be `p_breastFeeding` property with related properties inside it and `p_safeSpacing`  property with related properties inside it . `'patient' property is for first step basic info data